ansible/roles/stack-sunbird/tasks/service_stack.yml

```diff
@@ -2,14 +2,21 @@
 - name: Save stack file
   template: src="stack_{{service_name}}.yml" dest="/home/deployer/stack/stack_{{service_name}}.yml" mode=0644
 
+- name: Remove the existing config file
```

Comment:
you should have the file changed check. Because if we're doing simple deployment(without CDN for, say, variable changes) then we won't have index.js file, as we're cleaning the workspace. and we don't have to restart containers multiple times.

Reply:
you can use something like this